src/meetingsession/DefaultMeetingSession.ts

```diff
@@ -49,9 +46,14 @@ export default class DefaultMeetingSession implements MeetingSession, Destroyabl
 
     CSPMonitor.addLogger(this._logger);
     CSPMonitor.register();
-
-    this.setupEventReporter(configuration, logger, eventReporter);
-    this._deviceController = deviceController;
+    if (eventController) {
+      this._eventController = eventController;
+    } else {
+      this._eventController = new DefaultEventController(configuration, logger);
```

Comment:
We are creating multiple `DefaultEventController` objects, once here and one in `DefaultAudioVideoController`, this will lead to creating multiple `DefaultMeetingEventReporter` objects. Should we avoid this and use a single `DefaultEventController` across multiple classes in JS SDK?

Reply:
This actually won't create multiple because the meeting session will pass its eventController to the audioVideoController, but it's probably better to get rid of the default construction inside AudioVideo anyways, as it's a bit redundant.